Source/Core/Constants+LocalizationKeys.swift

```diff
@@ -0,0 +1,62 @@
+extension Constants {
+
+    struct LocalizationKeys {
+
+        struct BillingForm {
+            
+            struct AddressLine1 {
+                static let title = "addressLine1".localized
```

Comment:
This is fine but Since we expect the merchants to set any custom title we should also have a capability to allow merchants to pass whatever they want using their own localised string and what we do is just display.

Reply:
For now, any constant value is only internal to be used inside SDK for more clear values. 
The merchant can set any text when using BillingFormAddressLine1Style.
Constants will be public in the next ticket.

Reply:
I think @deepesh-vasthimal-cko has a point, that these values should be coming from consumer of SDK, not from the SDK itself. So instead of ever having to set a default value, the consumer cannot create the components without providing the texts.

In which case, these could be placed inside the sample app and even skip localisation altogether.

Reply:
Also, `LocalizationKeys` is not accurate as these are localisedStrings. Keys would be fine if it would return the key that the user of this structure would have to run `localised` against.

Reply:
Why does the internal SDK need these fields if we expect the end user to override anyway? Shouldn't we then save space and not add them if they're never to be seen? 

The options I understand:
- merchant doesn't have to provide these values, they are optional for them so we default to ours, in which case **do we cater for a large enough range of localisations from our side?**
- they are compulsory and you'll remove them in the next ticket? In which case, why are we adding them now, is it for testing?

Reply:
Constants will be public next ticket.

Reply:
I still don't follow. Would you please be able to describe to me the flow you'd see over the consumption of these Constants?

Reply:
@alex-ioja-yang-cko let's have a call to discuss it

Reply:
Thank you for the clarification on the merchant input being optional